src/main/java/org/adridadou/ethereum/EthereumContractInvocationHandler.java

```diff
@@ -114,8 +114,8 @@ private Constructor lookForNonEmptyConstructor(Class<?> returnType, Object[] res
         throw new IllegalArgumentException("no constructor with arguments found! for type " + returnType.getSimpleName());
     }
 
-    <T> void register(T proxy, Class<T> contractInterface, SoliditySource code, String contractName, EthAddress address, EthAccount account) {
-        final Map<String, CompilationResult.ContractMetadata> contractsFound = compile(code.getSource()).contracts;
+    <T> void register(T proxy, Class<T> contractInterface, SoliditySource code, String contractName, EthAddress address, EthAccount account) throws IOException {
+     	final Map<String, CompilationResult.ContractMetadata> contractsFound =  code.isCompiled() ?  CompilationResult.parse(code.getRawJson()).contracts :   compile(code.getSource()).contracts;
```

Comment:
This does not work well with the Standalone Blockchain(SB).
A quickfix would be adding a rawJson field to CompiledContract. 
But I will also look if there is another way to use CompiledContract together with the SB. (Wich means another patch)